ccdImports/configFiles/CCD_Probate_Backoffice/CaseField.json

```diff
@@ -64,16 +64,16 @@
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorFirmName", "Label": "Firm name", "FieldType": "Text", "SecurityClassification": "Public", "Min": 1, "Max": 100},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAddress", "Label": "Firm address", "FieldType": "AddressUK", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorEmail", "Label": "Solicitor email address", "FieldType": "Text", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorPhoneNumber", "Label": "Solicitor phone number", "FieldType": "Text", "SecurityClassification": "Public"},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAppReference", "Label": "Your reference", "FieldType": "Text", "SecurityClassification": "Public", "Max": 100},
-  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsExec", "Label": "Is the solicitor named in the will as an executor?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorFirmName", "Label": "Name of the firm, or the name under which the applicant operates", "FieldType": "Text", "SecurityClassification": "Public", "Min": 1, "Max": 100},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAddress", "Label": "Address of your firm or where the applicant operates", "FieldType": "AddressUK", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorEmail", "Label": "Contact email address", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorPhoneNumber", "Label": "Contact phone number", "FieldType": "Text", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorAppReference", "Label": "Your reference for this application", "FieldType": "Text", "SecurityClassification": "Public", "Max": 100},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsExec", "Label": "Is the qualified legal professional named in the will as an executor?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
+  {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorWillSignSOT", "Label": "Are you the qualified legal professional who will sign the statement of truth?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
   {"LiveFrom": "01/01/2017", "CaseTypeID": "GrantOfRepresentation", "ID": "solsSolicitorIsMainApplicant", "Label": "Is the solicitor the main applicant?", "FieldType": "YesOrNo", "SecurityClassification": "Public"},
```

Comment:
We should remove this field as well shouldn't we?

Reply:
Shall we do this in the general clean up ticket? I think it's still being used in the CW journey

Reply:
ok! CW journey needs to change too so will remove from there in the clean-up